datadog_checks_base/datadog_checks/base/utils/db/core.py

```diff
@@ -154,7 +156,7 @@ def execute(self, extra_tags=None):
                         submission_queue.append((transformer, value))
 
                 for transformer, value in submission_queue:
-                    transformer(sources, value, tags=tags)
+                    transformer(sources, value, tags=tags, hostname=self.hostname)
 
                 for name, transformer in query_extras:
                     try:
```

Comment:
You'll want to add `hostname=self.hostname` as well on line 161

Reply:
You are right, thanks! fixed